src/app/blog/[slug]/page.tsx

```diff
@@ -114,3 +135,42 @@ export async function generateMetadata({
     },
   };
 }
+
+const generateJsonLd = ({
```

Comment:
Let's convert this into a util and use the same function across the app.

Reply:
Yeah I've converted it to util in the structured-schema-generator file, seems it remained here. Removing it